### PR TITLE
Topic content term

### PR DESCRIPTION
**Pull request by thomasvlachos**
### Description

Created term-entry for .delay() in specified path as mentioned in description.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/d3/concepts/interactivity/terms/delay/delay.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+Title: '.delay()'
+Description: 'It introduces a waiting period before executing subsequent actions.'
```
Comment (avdhoottt) [inline content/d3/concepts/interactivity/terms/delay/delay.md:3]:
```suggestion
Description: 'Sets a waiting period before executing subsequent actions.' 
```

File: content/d3/concepts/interactivity/terms/delay/delay.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+The **`.delay()`** method is used within transitions to introduce a time delay before the execution of the specified actions.
+
+## Syntax
+
+```pseudo
+selection.transition()
+    .delay(delayValue)
+```
+The `.delay()` method is used in the context of transitions and sets the delay before the transition starts.
```
Comment (avdhoottt) [inline content/d3/concepts/interactivity/terms/delay/delay.md:24]: Add this line in the method introduction part.

File: content/d3/concepts/interactivity/terms/delay/delay.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+// Transition to change the radius with a delay
+circles.transition()
+    .delay(3000) // Introduce a delay of 3000 milliseconds (3 seconds)
+    .attr('r', 30); // Change the radius attribute during the transition
+```
+The example will result in the following output:
+
+
+![delay-3sec](https://github.com/thomasvlachos/docs/assets/32740965/b2e96749-638d-4903-8d95-3687c5236b36)
```
Comment (avdhoottt) [inline content/d3/concepts/interactivity/terms/delay/delay.md:48]: Read the [documentation](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#file-hosting) to add the media files properly and do it accordingly.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
Great Entry!! I have suggested some changes, please make them. Remove the extra line spaces from the end of the content.

**Review by thomasvlachos — COMMENTED** [review]
I'm not sure if this is the way to add media file. Inform me if i did something wrong to check it again. Thanks!

File: content/d3/concepts/interactivity/terms/delay/delay.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    .attr('cy', 100)
+    .attr('r', d => d)
+    .attr('fill', 'black');
+
+// Transition to change the radius with a delay
+circles.transition()
+    .delay(3000) // Introduce a delay of 3000 milliseconds (3 seconds)
+    .attr('r', 30); // Change the radius attribute during the transition
+```
+The example will result in the following output:
```
Comment (avdhoottt) [inline content/d3/concepts/interactivity/terms/delay/delay.md:45]: Give a line space after the code block.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@thomasvlachos I have suggested a minor change and the rest of the content is good. You've added the image properly, Thanks for doing it.

**Review by avdhoottt — APPROVED** [review]
Good for the second review.

**Review by dakshdeepHERE — APPROVED** [review]
This looks good to merge 🚀

**Pull request merged**